packages/coq/coq.8.3/opam

```diff
@@ -14,7 +14,7 @@ build: [
 ]
 depends: [
   "ocaml" {>= "3.10.2" & < "4.02.0"}
-  "camlp5" {!= "5.00"}
+  "camlp5" {!= "5.00" & < "8"}
```

Comment:
Wouldn't `>= 5` be better?

Reply:
Not with `>=`, as it would accept `5.00`. If it is compatible with versions `< 5`, then I'd rather keep the upper bound, otherwise I am happy to change to `> "5"`

Reply:
Ah right... maybe best to just not change it.^^